schema/formats_test.go

```diff
@@ -90,6 +90,83 @@ var _ = Describe("format checkers", func() {
 			result := formatChecker.IsFormat("::1/129")
 			Expect(result).To(Equal(false))
 		})
+
+		It("Should not pass - two zeros in octet IPv4", func() {
```

Comment:
Change all of these tests to Table tests. ref: https://onsi.github.io/ginkgo/#table-driven-tests

Reply:
this file is a bad example, but prefer test names that read like sentences. now it's "CIDR format checker / Should not pass - two zeros in octet IPv4". I suggest "CIDR format checker / should reject two zeros in octet IPv4" instead

Reply:
Good point. I changed descriptions to be more sentence like.